src/llmq/quorums.cpp

```diff
@@ -212,7 +212,7 @@ void CQuorumManager::EnsureQuorumConnections(Consensus::LLMQType llmqType, const
                             debugMsg += strprintf("  %s (%s)\n", c.ToString(), dmn->pdmnState->addr.ToString(false));
                         }
                     }
-                    LogPrint("llmq", debugMsg);
```

Comment:
thoughts on "casting" this way @UdjinM6 ?

Reply:
Sounds good imo

Reply:
There is another one https://github.com/dashpay/dash/blob/develop/src/llmq/quorums_dkgsessionhandler.cpp#L492

Reply:
resolved and force pushed